crates/shared/src/ethrpc/http.rs

```diff
@@ -93,14 +93,21 @@ async fn execute_rpc<T: DeserializeOwned>(
     // Log the raw text before decoding to get more information on responses that
     // aren't valid json. Debug encoding so we don't get control characters like
     // newlines in the output.
-    tracing::trace!(name = %inner.name, %id, body = %text.trim(), "received response");
```

Comment:
Can we leave the `trace` log here? I think it is useful (I still make use of it when debugging locally for example). It lets you see all RPC requests/responses that the services are making.

Reply:
Ofc. reverted.